docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  npm:
+    image: node:12
+    restart: "no"
+    volumes:
+      - .:/app
+    working_dir: /app
+    entrypoint: npm install
```

Comment:
This executes only once and runs the `npm install` command. This will generate the `node_modules` path locally.